ballerina/tests/test.bal

```diff
@@ -83,6 +84,45 @@ function testCreateObject() {
         test:assertFail(amazonS3Client.toString());
     }
 }
+@test:Config {
+    dependsOn: [testGetObject]
+}
+function testCreatePresignedURLGET() returns error? {
+    log:printInfo("amazonS3Client->createPresignedURLGET()");
+    Client|error amazonS3Client = new(amazonS3Config);
+    if (amazonS3Client is Client) {
+        string|error? response = amazonS3Client->createPresignedURL(testBucketName, fileName,"3600","GET");
+        if (response is error) {
+            test:assertFail(response.toString());
+        } else if (response is string) {
+            http:Client cl = check new(response);
+            http:Response a = check cl->get("");
+            test:assertEquals(a.statusCode, 200, "Failed to create presigned URL");
+        } else {
+            test:assertFail();
+        }
+    }
```

Comment:
If `amazonS3Client` is error this test will still pass? We need to handle the cases appropriately.